### PR TITLE
test2

### PR DESCRIPTION
**Pull request by grsky360**

### CODE REVIEW

**Review by chenscl — APPROVED** [review]

File: src/main/java/ilio/checks/ChecksApplication.java [hunk 1]
```diff
@@ -16,3 +16,4 @@ public void run(String... args) throws Exception {
         System.out.println("abc");
     }
 }
```
Comment (grsky360) [inline src/main/java/ilio/checks/ChecksApplication.java:18]: asdfasdf

**Review by chenscl — DISMISSED** [review]

**Review by chenscl — DISMISSED** [review]

**Review by chenscl — APPROVED** [review]

**Comment by grsky360** [comment]
asdf

File: src/test/groovy/ilio/checks/SimpleTest.groovy [hunk 2]
```diff
@@ -21,5 +21,6 @@ class SimpleTest extends Specification {
         a || RESULT
         1 || 1
         2 || 2
```
Comment (grsky360) [inline src/test/groovy/ilio/checks/SimpleTest.groovy:23]: asdf

File: gradlew.bat [hunk 3]
```diff
@@ -1,89 +0,0 @@
...
-@rem See the License for the specific language governing permissions and
-@rem limitations under the License.
-@rem
-
-@if "%DEBUG%" == "" @echo off
-@rem ##########################################################################
-@rem
-@rem  Gradle startup script for Windows
-@rem
-@rem ##########################################################################
```
Comment (grsky360) [inline gradlew.bat]: saf

**Comment by grsky360** [comment]
asdf

**Comment by grsky360** [comment]
asdf

**Comment by grsky360** [comment]
asdffdas

**Pull request merged**